### PR TITLE
IDEM-377

### PR DESCRIPTION
**Pull request by xiangzhe0812**
remove sync detail from gpib, fix gpib signup

### CODE REVIEW

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
Just what sonar qube said

File: app/src/hooks/userVendors.ts [hunk 1]
```diff
@@ -13,24 +13,18 @@ type UserInfo = {
...
-    vendorId: number
-  ) => Promise<void>;
 };
 
 const useVendors = (): Hooks => {
   const api = useApi();
 
   const signup = async (userInfo: UserInfo, vendorId: number) => {
-    const { name, email, mobile } = userInfo;
+    const { name, email, mobile, dob } = userInfo;
```
Comment (JasminFarhi) [inline app/src/hooks/userVendors.ts:27]: do we need DOB for GPIB?

Comment (xiangzhe0812) [reply app/src/hooks/userVendors.ts:27]: Need yob in gpib

Comment (bitcoinbrisbane) [reply app/src/hooks/userVendors.ts:27]: @xiangzhe0812 as discussed, lets post full DOB claim

**Review by brettvniekerk — APPROVED** [review]
LGTM when those comments are resolved

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**